content/en/docs/components/pipelines/v2/run-a-pipeline.md

```diff
@@ -1,80 +1,107 @@
 +++
 title = "Run a Pipeline"
-description = "Execute a pipeline on the KFP backend"
+description = "Run a pipeline"
 weight = 6
 +++
 
-The KFP SDK offers three ways to run a pipeline.
+You can run a pipeline in three ways:
 
-## 1. Run from the KFP Dashboard
-The first and easiest way to run a pipeline is by submitting it via the KFP dashboard.
+* [Run a pipeline from the KFP Dashboard](#run_from_dashboard)
+
+* [Run a pipeline from the KFP SDK client](#run_from_sdk_client)
+
+* [Run a pipeline from the KFP SDK CLI](#run_from_sdk_cli)
```

Comment:
The links for the second two bullets are not working for me. Can you check if they are for you? If not, perhaps Markdown-native section references (just linking to the title without HTML) will help with this?